.editorconfig

```diff
@@ -0,0 +1,13 @@
+# http://editorconfig.org
```

Comment:
At some point we might want these in the build tools? I'm not sure how to handle that, but all projects should share these settings.